packages/ui/atomic/create-atomic-component/index.js

```diff
@@ -1,35 +1,27 @@
 #!/usr/bin/env node
+import {
+  ensureComponentValidity,
+  camelize,
+  transform,
+  successMessage,
+} from '@coveo/create-atomic-commons';
 import '@coveo/create-atomic-component-project';
 import {dirname, resolve} from 'node:path';
-import {
-  cpSync,
-  renameSync,
-  unlinkSync,
-  writeFileSync,
-  readFileSync,
-} from 'node:fs';
+import {cpSync} from 'node:fs';
 import {cwd} from 'node:process';
 import {fileURLToPath} from 'node:url';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 const templateRelativeDir = 'template';
 const templateDirPath = resolve(__dirname, templateRelativeDir);
 
-const camelize = (str) =>
-  str
-    .replace(/-(.)/g, (_, group) => group.toUpperCase())
-    .replace(/^./, (match) => match.toUpperCase());
-
 cpSync(templateDirPath, cwd(), {
   recursive: true,
 });
 
 let componentName = process.argv[2];
 if (componentName) {
-  if (!componentName?.includes('-')) {
-    componentName = `atomic-${componentName}`;
-  }
-
+  ensureComponentValidity(componentName);
```

Comment:
Throws (with a nice message :) ) whenever the component name does not respect HTML standards